### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by seanherron**
This adds a significant amount of documentation and developer support:
- Uses `mkdocs` and Read the Docs to provide rich documentation
- Increases Vagrant developer performance

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +31,7 @@ $ vagrant ssh
 $ python manage.py runserver 0.0.0.0:8000
 ```
 
-You can then access the site from your web browser by going to http://localhost:8000
+You can then access the site from your web browser by going to http://192.168.19.61:8000
```
Comment (cmc333333) [inline README.md:34]: This specific IP is dependent on the user; mine would be different than yours.

Comment (cmc333333) [inline README.md:34]: Welp. I needed to read further.

File: docs/docs/about.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# hello
```
Comment (cmc333333) [inline docs/docs/about.md]: Should this be included?

File: README.md (hunk 1 above)
Comment (seanherron) [inline README.md:34]: READ THE DOCS YO

File: docs/docs/about.md (hunk 2 above)
Comment (seanherron) [inline docs/docs/about.md]: fixed

File: docs/docs/developer-guide/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+A complete walk-though of `git` is outside the scope of the documentation, but follows is a brief overview of concepts on how we manage the repository and releases.
+
+## Developer Workflow
+
+- Each developer works off a `fork` of the main [18F Repository](https://github.com/18f/peacecorps-site). These forks are located in their personal GitHub accounts. For instance, here is [Sean Herron's](https://github.com/seanherron/peacecorps-site) and [CM Lubinski's](https://github.com/cmc333333/peacecorps-site).
+- Individual Feature Enhancements, bugfixes, or changes are created as `branches` off the main `master` branch. For instance, if a developer is creating a feature that introduces search across the site, they would create a `search` branch of `master` which contains all work specific to search. If they, at the same time, needed to fix a bug unrelated to search (perhaps around font sizes), they would `commit` their work in `search`, switch back to `master`, and create a new `fix_font_size` branch with that specific bugfix. The goal is to scope each branch to a specific item of work, and not bundle too many things together.
```
Comment (cmc333333) [inline docs/docs/developer-guide/getting-started.md]: branch of `master` -> branch off `master`

File: docs/docs/user-guide/media.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+![Add Block](images/new_text_block.png)
+
+Selecting `image` will create a new image block, to support inline images in text:
+
+![Add Block](images/new_image_block.png)
+
+Provide a **title**, **description**, and select **choose a file** to upload an image file from your computer. Images should be properly cropped and compressed. We use [ImageOptim](https://imageoptim.com/) for this.
+
+### Deleting Moving Blocks
```
Comment (cmc333333) [inline docs/docs/user-guide/media.md]: Deleting and Moving*

File: docs/docs/user-guide/pcpp.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+### Projects and Campaigns
+The other entry that is automatically created depends on the type of account. PCPP Projects receive an entry in `Projects`, and everything else (eg. Memorial Funds, Country Funds, Sector Funds, etc) receive an entry in `Campaigns`. 
+
+Data within `Projects` and `Campaigns` is _initially populated_ by data from Odyssey, but any changes after the initial population will not be transferred from Odyssey to the CMS. You'll need to make all edits in `Projects` and `Campaigns` within the CMS in order for them to take effect.
+
+For more information, see [Creating a New PCPP Project](#creatingediting-pcpp-projects) and [Creating a New PCPP Campaign](#creatingediting-pcpp-campaigns).
+
+## Issues
+
+Campaigns and Projects also may belong to an "Issue". An issue is a broad categorization of themes, for instance, "Agriculture", "Health", or "Education". They don't need to map to specific sectors, however, each project or campaign **must** belong to at least one issue. Issues are primarily used in the sorter to help with project navigation:
```
Comment (cmc333333) [inline docs/docs/user-guide/pcpp.md:35]: Important to note that, in this code base, projects are associated to issues _through_ campaigns. i.e. campaigns belong to issues, and projects are associated with campaigns, but there is no direct link between projects and issues.

File: docs/docs/developer-guide/deployment.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
+<h1>Deployment</h1>
```
Comment (cmc333333) [inline docs/docs/developer-guide/deployment.md:1]: This doesn't explicitly reference peace-corps-infrastructure repo, which it probably should

**Comment by cmc333333** [comment]
This is great, @seanherron ! Do you think it should mention nginx, gunicorn, and application logs? Combined, those make debugging failure much easier.

**Comment by seanherron** [comment]
@cmc333333 pushed up changes that should reflect that input

**Pull request merged**